src/components/card-form.component.scss

```diff
@@ -0,0 +1,26 @@
+@import "../globals/variables.scss";
+
+.card-form {
+  @include card;
+  width: 400px;
+  height: 553px;
+  position: absolute;
+  z-index: 3;
+  right: 40%;
```

Comment:
### ⚠️The most important thing to learn in the PR!

It isn't the best way to center elements.
There are several methods how toy can crack the case, you can find the details [here](https://skillbox.ru/media/code/vyravnivanie-po-tsentru-v-css-priyemy-svoystva-podvodnye-kamni/)

In a nutshell, there are two common approaches:
- Set `flexbox` + `justify-content: center` + `align-items: center` to the parent element;
OR
- Set `position: absolute` + `top and left: 50%` + `transform: translate(-50%, -50%)` to the centered element

If you're not familiar with the css property `transform` - watch the [video](https://www.youtube.com/watch?v=rzD-cPhq02E), it'll help a lot

Reply:
Wow) Watched and learned this property) Worked out fine!